components/src/dynamo/triton/tests/conftest.py

```
@@ -0,0 +1,79 @@
+_tritonserver_importable: bool | None = None
+
+
+def _can_import_tritonclient() -> bool:
+    """Import the worker's tritonclient deps once and cache the result."""
+    global _tritonclient_importable
+    if _tritonclient_importable is None:
+        try:
+            importlib.import_module("tritonclient.grpc.model_config_pb2")
+            importlib.import_module("tritonclient.utils")
+            _tritonclient_importable = True
+        except Exception:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)